core/src/main/resources/core-migrations/local/V3__Logging.sql

```diff
@@ -4,4 +4,50 @@ create table if not exists logs
     Level     varchar(255)                          not null,
     Message   text                                  not null,
     Timestamp timestamp default current_timestamp() not null
-);
\ No newline at end of file
+);
+
+create table if not exists uuiditems
+(
+    UUID        varchar(255)    PRIMARY KEY,
+    Namespace   varchar(255)    NOT NULL,
+    Keyname     varchar(255)    NOT NULL
+);
+
+create table if not exists uuidlogmeta
+(
+    id      int auto_increment  not null,
+    logId   int                 not null,
+    UUID    varchar(255)        not null,
+    Type    varchar(255)        not null,
+    Player  varchar(255)        default null,
```

Comment:
Originally setup differently, I did want to also include Clan UUID, but that isn't super important.
I am considering adding location/world, all current messages have that in them, and would allow for some interesting database searches.

Reply:
Can you include some screenshots of the data set, am curious to see what it looks like

Reply:
You could also have a column for UUID type, lets you avoid creating a Clan UUID column, and supports more types in the future

Reply:
> Can you include some screenshots of the data set, am curious to see what it looks like

https://prnt.sc/RmwNyi6Au5dh
https://prnt.sc/AOeG0N_z7fR2
https://prnt.sc/0SDmR9mw-YcI

Reply:
> You could also have a column for UUID type, lets you avoid creating a Clan UUID column, and supports more types in the future

I think this approach makes the most sense